src/BmiForm.jsx

```diff
@@ -38,6 +38,8 @@ const BmiForm = ({ change }) => {
             name="weight"
             type="number"
             maxLength="3"
+            min="1" 
+            max="999"
```

Comment:
Still not working, I am able to add more than 3 digits.
I think I also tried "number" thing in past but it didn't work out.